setup.py

```diff
@@ -126,10 +126,9 @@ def read_requirements():
 # read version from the package file.
 def read_version():
     version_str = '1.0.0'
-    with (open(os.path.join(TOP_DIR, 'onnxruntime_extensions/_version.py'), "r")) as f:
-        line = [_ for _ in [dedent(_) for _ in f.readlines()] if _.startswith("__version__")]
-        if len(line) > 0:
-            version_str = line[0].split('=')[1].strip('" \n\r')
+    with (open(os.path.join(TOP_DIR, 'version.txt'), "r")) as f:
+        line = [_ for _ in [dedent(_) for _ in f.readlines()]]
```

Comment:
could it be as simple as:
```suggestion
        version_str = f.readline().strip()
```